src/sampling/homogeneous.jl

```diff
@@ -40,6 +40,24 @@ function sample(rng::AbstractRNG, triangle::Triangle{Dim,T}, method::Homogeneous
   (randpoint() for _ in 1:(method.size))
 end
 
+function sample(rng::AbstractRNG, ball::Ball{Dim,T}, method::HomogeneousSampling) where {Dim,T}
+  if Dim == 2
```

Comment:
Let's use multiple-dispatch with Dim for better clarity. That is, two method definitions one for Ball{2,T} and another for Ball{3,T}